docs/style.css

```
@@ -17,7 +17,8 @@ h1 {
 }
 
 .center {
-    margin: var(--section-gap) auto;
+    padding-top: var(--section-gap);
```

It doesn't take into account space in the bottom of the page. This should fix this.

```suggestion
    padding: var(--section-gap) 0;
```